docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,142 @@
+[Run a Non-validator Node](non-validator-node.mdx), and [Run a ParaTime client Node](paratime-client-node.mdx)
+sections and have a working ParaTime client node with [TEE support].
+
+### Stake Requirements
+
+To be able to register as a ParaTime observer node on the Oasis Network, you need to have enough tokens staked in your
+entity's escrow account.
+
+Current minimum staking requirements for a specific ParaTime are listed on the [Stake Requirements] page. You can also
+use [Oasis CLI tools] to check that as described in [Common Staking Info].
+
+Finally, to stake the tokens and to check if you staked correctly, you can use any wallet and any explorer. If using
+our [Oasis CLI tools] and if everything was set up correctly, you should see something like below when running
+[`oasis account show`] command for your entity's account (this is an example for Testnet):
+
+![code shell](../../../external/cli/examples/account/show-delegations.in.static)
+
+![code](../../../external/cli/examples/account/show-delegations.out.static)
+
+:::caution
+
+The stake requirements may differ from ParaTime to ParaTime and are subject to change in the future.
+
+Currently, for example, if you want to register an observer node for Testnet/Mainnet, you currently need to have at
+least **200 TEST/ROSE** tokens delegated:
+- **100 TEST/ROSE** for registering a new node entity and,
+- **100 TEST/ROSE** for gaining an observer role.
+
+:::
+
+[Stake Requirements]: prerequisites/stake-requirements.md
+[Run a ParaTime Node]: ../../get-involved/run-node/paratime-node.mdx
+[Common Staking Info]: ../../general/manage-tokens/cli/network.md#show-native-token
+[TEE support]: prerequisites/set-up-trusted-execution-environment-tee.md
+[Oasis CLI tools]: ../../general/manage-tokens/cli/account.md#delegate
+[`oasis account show`]: ../../general/manage-tokens/cli/account.md#show
+
+### Register a New Entity or Update Your Entity Registration
+
+:::danger
+
+Everything in this section should be done on the `localhost` as there are sensitive items that will be created.
+
+:::
+
+:::danger
+
+If you plan to run an observer node for Mainnet and Testnet make sure you create and use two separate entities to
+prevent replay attacks.
+
+:::
+
+1. If you don't have an entity yet, create a new one by following the [Initialize Entity] instructions for validators.
+
+2. If you will be running the ParaTime on a new Oasis node, also initialize a new node by following the
+   [Starting the Oasis Node] instructions for validators.
+
+3. Now, [list your node ID] in the entity descriptor file `nodes` field.
+
+4. [Register] the updated entity descriptor.
+
+5. By adding the created entity ID in the node config file, you will [configure the node] to automatically register for
+   the roles it has enabled (i.e. observer role) via the `registration.entity_id` configuration flag. No manual node
+   registration is necessary.
+
+   ```yaml
+   # ... sections not relevant are omitted ...
```

Comment:
```suggestion
   mode: client
   # ... sections not relevant are omitted ...
```
Lets make it obvious that this has the mode is still `client`.

Reply:
Refactored.